crates/wkg/src/oci.rs

```diff
@@ -36,16 +36,20 @@ impl Auth {
         match (self.username, self.password) {
             (Some(username), Some(password)) => Ok(RegistryAuth::Basic(username, password)),
             (None, None) => {
-                let server_url = format!("https://{}", reference.registry());
-                match docker_credential::get_credential(&server_url) {
-                    Ok(DockerCredential::UsernamePassword(username, password)) => {
-                        return Ok(RegistryAuth::Basic(username, password));
-                    }
-                    Ok(DockerCredential::IdentityToken(_)) => {
-                        return Err(anyhow::anyhow!("identity tokens not supported"));
-                    }
-                    Err(err) => {
-                        tracing::debug!("Failed to look up OCI credentials: {err}");
+                let server_url = format!("https://{}", reference.resolve_registry());
+                for server_url in [&server_url, &format!("{server_url}/v1/")] {
```

Comment:
Is docker the only one that forces the `v1` ending? If so, it might be clearer to just check if it is docker and then append. A comment here would also clarify. Took me a second when I first looked

Reply:
I've looked into the sources of `docker login` and did not find any other cases where `/v1/` would be appended, also according to [this](https://jfrog.com/help/r/jfrog-artifactory-documentation/test-your-docker-and-artifactory-cloud-setup):
>You cannot use a context path when providing the registry path (e.g localhost:8082/artifactory is not valid)

So I've replaced the for loop with a 1:1 mapping as you suggested.